extensions/low-power-mode/src/utils.ts

```diff
@@ -0,0 +1,25 @@
+import { runAppleScript, showFailureToast } from "@raycast/utils";
+import { showHUD } from "@raycast/api";
+
+import { execSync } from "child_process";
+
+export const isLowPowerModeEnabled = () => {
+  try {
+    const result = execSync("pmset -g | grep lowpowermode | awk -F' ' '{ print$2 }'");
+    const resultInString = result.toString().trim();
```

Comment:
This always returns blank string for me in Sonoma, it doesn't work as expected for me

Reply:
That's strange. I am using macOS Sonoma too.

I tried this command for both `Admin` user and a `Standard` user. 
This command returns value for both kinds of user.

Not sure what other difference could have caused this command to return blank string.

<img width="800" alt="image" src="https://github.com/raycast/extensions/assets/5305600/e6819a80-e65f-4374-aaf7-2d8bb4a65325">

Reply:
I have updated this PR with an alternate mode to get the Low Power Mode status.

https://github.com/raycast/extensions/pull/9227/commits/cdadf16bb9786f020b6b4fce26ac539412ca5021

Kindly check if this works for you.

Reply:
I don't have that in `pmset`

![Raycast-99u9vHFi@2x](https://github.com/raycast/extensions/assets/38033107/3ef3c405-7cc7-4a27-a614-84dd1177cce5)

Reply:
Are you on a M1/2/3 Max MacBook Pro? Those have an option for "high power mode". Instead of having `lowpowermode` in `pmset`, it's just called `powermode`. Seems like `0`=automatic, `1`=low, `2`=high ([Source](https://apple.stackexchange.com/questions/452488/how-can-i-set-the-low-power-mode-to-only-on-battery-programmatically))

Reply:
Yes, I'm on M2 🙂

Reply:
Here's what I see.

![CleanShot 11-29 at 11 41 57](https://github.com/raycast/extensions/assets/43297314/a28a1463-8655-4065-9e11-dba764a8e808)